### ISSUE TITLE
Построение сводов по документам.

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1715]
Originally reported on Google Code with ID 1715

```
У нас своды строятся из итогового начисления. При построении этим способом
есть главный недостаток. Т.к. есть временные переводы и люди работают не по
своему основному месту работы.  Еще есть проблема с выбором нужных нам
людей в итоговое  для построения свода  при этом пересчитать только их. А
затем нам надо пересчитать только тех людей,  которые нам нужны для
построения свода (из разных подразделений), что крайне неудобно.  
Т.к. готова настройка, в которой все  виды нужные нам для свода считаются в
3-х основных документах (Бригадный наряд, начисление повременно и сдельно)
и в каждом документе в шапке указывается подразделение, то предлагается
строить свод по этим документам.

```

Reported by `roberto1813` on 2009-11-12 13:28:59

**Comment by gsbelarus** [comment]
```
Свод не возможно построить по данным документам, и вы знаете почему (т.к. в данный 
момент этих начислений физически не существует).
Я могу сделать, что документы (которые вы перечислите. будут заносить данные в 
журнал сами, без итогового начисления (как бы предварительно)), тогда можно будет 
построить данные отчёт. Единственный ньюанс, что сотрудники должны быть всё равно в

итоговом начислении.
```

Reported by `Alexander.GoldenSoft` on 2009-11-12 13:38:56

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-11-12 13:44:17
- **Labels added**: Wage, Gomel
- **Labels removed**: WageGomel

**Comment by gsbelarus** [comment]
```
ну если это самый лучший вариант, то тогда надо чтоб авансом попадали, чтоб без
пересчета можно было бы строить свод, как ты предлагаешь.
```

Reported by `roberto1813` on 2009-11-12 13:54:54

**Comment by gsbelarus** [comment]
```
завтра будет :)
```

Reported by `Alexander.GoldenSoft` on 2009-11-12 15:09:19
- **Status changed**: `Started`

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-11-12 17:59:52
- **Labels added**: GMK
- **Labels removed**: Gomel

**Comment by gsbelarus** [comment]
```
Вопрос с временными перемещениями поставлен глобально здесь: 
http://code.google.com/p/gedemin/issues/detail?id=1900

Вопрос по поводу фильтра в итоговом по подразделению решен на GMK. Аналогичный 
фильтр понадобится в стандарте, если мы добавим в шапки Начисления по окладу, 
Начисления за повременную работу, Бригадного наряда поле Подразделение. Отдельно 
вынесено в http://code.google.com/p/gedemin/issues/detail?id=1901
```

Reported by `alexandra.gsoftware` on 2010-03-03 12:08:31
- **Status changed**: `Duplicate`
- **Labels added**: Priority-Medium
- **Labels removed**: Priority-High
- **Merged into**: #1900
